remix-lib/src/web3Provider/web3VmProvider.js

```diff
@@ -97,7 +97,9 @@ web3VmProvider.prototype.txWillProcess = function (self, data) {
 
 web3VmProvider.prototype.txProcessed = function (self, data) {
   var lastOp = self.vmTraces[self.processingHash].structLogs[self.processingIndex - 1]
-  lastOp.error = lastOp.op !== 'RETURN' && lastOp.op !== 'STOP'
+  if (lastOp) {
+    lastOp.error = lastOp.op !== 'RETURN' && lastOp.op !== 'STOP'
```

Comment:
Shouldn't this also contain `SELFDESTRUCT`? That supposed to abort execution.

Reply:
yes